### PR TITLE
Add ids filter subscription

### PR DESCRIPTION
**Pull request by AnthonySendra**
Add realtime ids filter

```js
kuzzle
  .dataCollectionFactory('user')
  .subscribe({
    ids: {
      values: ['myDocumentId']
    }
  })

### CODE REVIEW

File: lib/api/core/models/requestObject.js [hunk 1]
```diff
@@ -35,10 +35,13 @@ function construct (object, additionalData, protocol) {
   if (object.body && object.body._id !== undefined) {
```
Comment (scottinet) [inline lib/api/core/models/requestObject.js]: This means that we take the first found document ID, in this order: document body => document headers => additional data

I wonder if this shouldn't be the other way around: first the _id present in additional data, if any, then the one in the headers, and lastly, if no other _id was found, the one in the document body.

What do you think?

**Comment by benoitvidis** [comment]
+1

**Comment by jenow** [comment]
+1

**Comment by scottinet** [comment]
+1

**Pull request merged**